packages/core/src/test/codewhisperer/commands/basicCommands.test.ts

```diff
@@ -477,7 +451,7 @@ describe('CodeWhisperer-basicCommands', function () {
             getTestWindow().onDidShowQuickPick(async (e) => {
                 e.assertItems([
                     createSeparator('Inline Suggestions'),
-                    createAutoSuggestions(false),
+                    createAutoSuggestions(true),
```

Comment:
`true` is the default since https://github.com/aws/aws-toolkit-vscode/commit/2fed6a62288c291d22b6555abd4b23569fc31c3e . This wasn't caught because the tests use stubs/mocks. This is an example of why stubs/mocks are fragile and don't represent the actual application code.